scriptTest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+if [ "$1" = "install" ]; then
+	clear
+	echo "=========== Begin to install and build ========="
+	sudo python3.4 setup.py install
+	sudo python3.4 setup.py build
+	echo "=========== Installed and built the sniffles ========="
+	echo
+elif [ "$1" = "test" ]; then
+	clear
+	echo "=========== Begin to install and build ========="
+	sudo python3.4 setup.py install
```

Comment:
if you run ./scriptTest install, it will reinstall everything but it does not run the test cases
if you run ./scriptTest test, it will reinstall everything then it will execute the test cases